src/System.IO.Compression.Brotli/System/IO/Compression/Brotli/BrotliPrimitives.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Text;
+
+#if BIT64
+    using nuint = System.UInt64;
+#else
+using nuint = System.UInt32;
```

Comment:
Yes, it is interesting. I just have done Ctrl-A, Ctrl-K, Ctrl+F and it changed, but it doesn't look like standard style.